src/modules/physics/physics.h

```diff
@@ -139,7 +139,7 @@ void lovrColliderSetMassData(Collider* collider, float cx, float cy, float cz, f
 void lovrColliderGetPosition(Collider* collider, float* x, float* y, float* z);
 void lovrColliderSetPosition(Collider* collider, float x, float y, float z);
 void lovrColliderGetOrientation(Collider* collider, float* angle, float* x, float* y, float* z);
-void lovrColliderSetOrientation(Collider* collider, float angle, float x, float y, float z);
+void lovrColliderSetOrientation(Collider* collider, dQuaternion q);
```

Comment:
Maybe a minor thing, but I try to avoid library-specific types in the function signatures.  So I would prefer using a `float*` / `quat` type here.